client/src/containers/interventions/list/component.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+
+import { ScenarioInterventionsGrowthItems } from 'containers/scenarios/types';
+
+const InterventionsList: FC<ScenarioInterventionsGrowthItems> = ({
+  items,
+}: ScenarioInterventionsGrowthItems) => {
+  return (
+    <>
+      {!items.length && (
```

Comment:
be careful with this because it will render a `0`
```suggestion
      {items.length === 0 && (
```